### PR TITLE
massaging the discussion

### PR DESCRIPTION
**Pull request by petrelharp**
Here's my proposed edits for #80; if they look good I will push them to that PR (I think I can do this).

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
...
-the \stdpopsim library. We have founded the Consortium with a number of specific goals
-in mind: standardization of simulation within the computational genetics community,
-increasing reproducibility of simulation based results, community-based development and
-decision making in guiding best practices in population genetics, and eventual benchmarking
-of inferential methods.
+the \stdpopsim library. We have founded the Consortium with a number of specific goals in mind:
+standardization of simulation within the computational genetics community,
+increasing reproducibility and ease of use of complex simulation based results,
+community-based development and decision making guiding best practices in population genetics,
+and eventual benchmarking of inferential methods.
```
Comment (petrelharp) [inline popSimManu.tex:526]: just adding "ease of use" here

File: popSimManu.tex [hunk 2]
```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
...
 
-Moreover individual research groups are often in the position of having to independently
-reimplement complex previously published models. \stdpopsim offers solutions to both of
-these issues by creating a quality controlled, easy-to-use, gateway to models and organisms
+Individual research groups are often in the position of having to independently
+reimplement complex previously published demographic models.
+Additional layers of realism (e.g., recombination maps) makes this especially daunting.
+\stdpopsim aims to lower this barrier
+by creating a quality controlled, easy-to-use, gateway to models and organisms
 that are central to modern research in genetics.
```
Comment (petrelharp) [inline popSimManu.tex:541]: trying to use the phrase 'lower the barrier'

File: popSimManu.tex [hunk 3]
```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
...
-tends to over-estimate oscillations in size.  By contrast, in the case of
-\emph{D.~melanogaster}, where linkage disequilibrium is considerably
-reduced, \stairwayplot outperforms the sequence-based methods
-(Figure~\ref{fig:n_t_sheehan}). In the two-population setting for human
+(Figures~\ref{fig:n_t_ragsdale} and~\ref{fig:n_t_gutenkunst}), which
+tends to over-estimate the size of oscillations.  By contrast,
+\stairwayplot outperforms the sequence-based methods
+on simulations of \textit{D.~melanogaster} populations,
+in which linkage disequilibrium is considerably reduced (Figure~\ref{fig:n_t_sheehan}).
+In simulations of two human populations
```
Comment (petrelharp) [inline popSimManu.tex:556]: smoothing over the somewhat confusing wording here

File: popSimManu.tex [hunk 4]
```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
...
 (Figure~\ref{fig:IM_popn_human}), most methods do reasonably well at
 reconstructing the YRI history, but struggle with the more complex CEU
 history, in large part because of the restriction of constant sizes per
 population.  An exception is \smcpp, which does not have the same
 restrictions on its inferred history, and as a result does somewhat better
 with the CEU history but tends to overfit the YRI history.  The results for
-the two-population {\em Drosophila} case (Figure~\ref{fig:two_popn_fly})
+the two-population \textit{D.~melanogaster} case (Figure~\ref{fig:two_popn_fly})
 are generally similar.
```
Comment (petrelharp) [inline popSimManu.tex:564]: technically species names are italicised not emphasized; and, we used Dmel above, so should be consistent.

File: popSimManu.tex [hunk 5]
```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
...
 Moreover the comparison of methods across the various genome organizations, genetic maps,
 and demographic histories of different organisms, provides valuable information
 about how methods might perform on non-human systems.
 
 Finally, comparison of results across methods or simulation runs
 provides an estimate of inference uncertainty, analogous to parametric
-bootstrapping.
+bootstrapping,
+especially since different methods are likely vulnerable to model misspecification
+in different ways.
```
Comment (petrelharp) [inline popSimManu.tex:580]: clarifying what we mean here by 'bootstrapping' (across runs is like bootstrapping, but so is using different methods, although less clearly)

File: popSimManu.tex [hunk 6]
```diff
@@ -519,68 +519,73 @@ \subsection*{Use case: comparing methods of demographic inference}
...
 Our implementations of genome representations and genetic maps for the some of
-the most common study systems in computational genetics---humans, \emph{Drosophila},
-and \emph{Arabidopsis} (among others)---are only intended to be a starting point for
-future development.  Indeed we envision future work to incorporate common modeling
-violations such as selection, gene conversion, and mutational heterogeneity, 
+the most common study systems in computational genetics---humans, \textit{Drosophila},
+and \textit{Arabidopsis} (among others)---are only intended to be a starting point for
+future development.  In addition to other taxa,
+we hope to soon incorporate other common biological processes
+such as selection, gene conversion, and mutational heterogeneity,
```
Comment (petrelharp) [inline popSimManu.tex]: selection is *biology*, not a model violation =)

File: popSimManu.tex (hunk 4 above)
Comment (andrewkern) [reply popSimManu.tex:564]: I wonder if we should just use the `stdpopsim` IDs e.g., DroMel

File: popSimManu.tex (hunk 6 above)
Comment (andrewkern) [inline popSimManu.tex]: doesn't quite work with the rest of this sentence. rephrase?

File: popSimManu.tex (hunk 2 above)
Comment (andrewkern) [reply popSimManu.tex:541]: we are missing a comma between "complex" and "previously published", but we can fix that later

Comment (petrelharp) [reply popSimManu.tex:541]: not sure I agree but either way

File: popSimManu.tex (hunk 4 above)
Comment (petrelharp) [reply popSimManu.tex:564]: only if you use HomSap above (I vote keep the organism name tho)

File: popSimManu.tex (hunk 6 above)
Comment (petrelharp) [reply popSimManu.tex]: oh whoops

Comment (petrelharp) [reply popSimManu.tex]: fixed

**Comment by andrewkern** [comment]
okay rebase and let's merge this

**Comment by andrewkern** [comment]
hey don't you want to PR this into `popsim-consortium/master` ?

**Comment by jeromekelleher** [comment]
You need to merge this @andrewkern, and then it'll show up on the PR going in to master.

**Comment by andrewkern** [comment]
i'm a bit confused by this network. Do I need to rebase this before I merge? Can I do it after?

**Comment by andrewkern** [comment]
nvm-- i figured it out

**Pull request merged**